Nexus-Docs/Versions.md

```diff
@@ -0,0 +1,26 @@
+## LifeHashes In Five Flavors
+
+LifeHashes can be displayed in one of five ways (one of which is deprecated:
+
+* **`version1`** The Original. **DEPRECATED**
+* **`version2`** Bug fixes and CMYK-friendly.
```

Comment:
I would add that this is the "official" LifeHash version.

Reply:
Noted.